src/modules/UI.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable linebreak-style */
+/* eslint-disable prefer-const */
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] As the previous reviewer said, kindly fix also this part. set up linters locally and run this command `npx eslint . --fix`